src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -114,6 +116,9 @@ public function load(array $configs, ContainerBuilder $container)
         // load ACL
         if (isset($config['acl'])) {
             $this->aclLoad($config['acl'], $container);
+        } else {
+            $container->removeDefinition(InitAclCommand::class);
```

Comment:
not sure.. removeDefinition is non-fatal. Did this for translations, routing, etc. as well.